### ISSUE TITLE
1.17|  I don´t know what issue is exactly but get Failed to add PDH Counter and Failed to read console input

### ISSUE THREAD

**Issue by broken1arrow** [PaperMC/Paper#5853]
### Expected behavior

Not get this error. I know what it is, but not what it cause it.

```
[16:45:39] [main/WARN]: [oshi.util.platform.windows.PerfDataUtil] Failed to add PDH Counter: \V�xlingsfil(_Total)\% Usage, Error code: 0xC0000BB8
[16:45:39] [main/WARN]: [oshi.util.platform.windows.PerfCounterQueryHandler] Failed to add counter for PDH object: V�xlingsfil
[16:45:39] [main/WARN]: [oshi.util.platform.windows.PerfCounterQuery] Disabling further attempts to query Paging File.
```

### Observed/Actual behavior

That I show above.

### Steps/models to reproduce

Start the server trigger this.

### Plugin list

null

### Paper version

Paper-1.17-R0.1-EXPERIMENTAL.jar

### Agreements

- [X] I am running the latest version of Paper available from https://papermc.io/downloads.
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] My version of Minecraft is supported by Paper.

### Other

And get even this error, if I use the gui.

```
[20:28:18] [Server console handler/ERROR]: [TerminalConsole] Failed to read console input
java.io.IOException: Referensen (handle) �r felaktig
	at java.io.FileInputStream.readBytes(Native Method) ~[?:?]
	at java.io.FileInputStream.read(FileInputStream.java:271) ~[?:?]
	at java.io.BufferedInputStream.read1(BufferedInputStream.java:282) ~[?:?]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:343) ~[?:?]
	at sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:297) ~[?:?]
	at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:339) ~[?:?]
	at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:188) ~[?:?]
	at java.io.InputStreamReader.read(InputStreamReader.java:178) ~[?:?]
	at java.io.BufferedReader.fill(BufferedReader.java:161) ~[?:?]
	at java.io.BufferedReader.readLine(BufferedReader.java:329) ~[?:?]
	at java.io.BufferedReader.readLine(BufferedReader.java:396) ~[?:?]
	at net.minecrell.terminalconsole.SimpleTerminalConsole.readCommands(SimpleTerminalConsole.java:180) ~[patched_1.17.jar:git-Paper-"3740db0"]
	at net.minecrell.terminalconsole.SimpleTerminalConsole.start(SimpleTerminalConsole.java:143) ~[patched_1.17.jar:git-Paper-"3740db0"]
	at net.minecraft.server.dedicated.DedicatedServer$1.run(DedicatedServer.java:112) ~[patched_1.17.jar:git-Paper-"3740db0"]
	Suppressed: java.io.IOException: Referensen (handle) �r felaktig
		at java.io.FileDescriptor.close0(Native Method) ~[?:?]
		at java.io.FileDescriptor.close(FileDescriptor.java:297) ~[?:?]
		at java.io.FileInputStream$1.close(FileInputStream.java:366) ~[?:?]
		at java.io.FileDescriptor.closeAll(FileDescriptor.java:355) ~[?:?]
		at java.io.FileInputStream.close(FileInputStream.java:364) ~[?:?]
		at java.io.BufferedInputStream.close(BufferedInputStream.java:481) ~[?:?]
		at sun.nio.cs.StreamDecoder.implClose(StreamDecoder.java:391) ~[?:?]
		at sun.nio.cs.StreamDecoder.close(StreamDecoder.java:204) ~[?:?]
		at java.io.InputStreamReader.close(InputStreamReader.java:193) ~[?:?]
		at java.io.BufferedReader.close(BufferedReader.java:532) ~[?:?]
		at net.minecrell.terminalconsole.SimpleTerminalConsole.readCommands(SimpleTerminalConsole.java:183) ~[patched_1.17.jar:git-Paper-"3740db0"]
		at net.minecrell.terminalconsole.SimpleTerminalConsole.start(SimpleTerminalConsole.java:143) ~[patched_1.17.jar:git-Paper-"3740db0"]
		at net.minecraft.server.dedicated.DedicatedServer$1.run(DedicatedServer.java:112) ~[patched_1.17.jar:git-Paper-"3740db0"]
```

**Comment by broken1arrow** [comment]
If you need some translations, for words some are on different language an English.

V�xlingsfil = swap file /paging file,
�r felaktig = are wrong/are incorrect.
Referensen = The reference

**Comment by kennytv** [comment]
https://bugs.mojang.com/browse/MC-228430 the fix is as simple as bumping the dependency, not sure if it's even worth making a patch before Mojang releases the first 1.17 patch release.

**Comment by broken1arrow** [comment]
Nice so 1.17.1 is confirmed :) 

Is not cause bigger issue (like big delay as for him).

**Comment by kennytv** [comment]
Fixed in https://github.com/PaperMC/Paper/commit/ac95f25f3534d0c5e65ea7d106973639a58670a1 since the extra startup delay started to annoy me as well (we should just remember to remove the patch in 1.17.1).

**Comment by broken1arrow** [comment]
> Fixed in [ac95f25](https://github.com/PaperMC/Paper/commit/ac95f25f3534d0c5e65ea7d106973639a58670a1) since the extra startup delay started to annoy me as well (we should just remember to remove the patch in 1.17.1).

I can try to remember, so can I send a notice when mojang has fix it :)
